integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -1,11 +1,13 @@
 set -e
 
+HBASE_VERSION=$(curl https://apache.mirror.gtcomm.net/hbase/stable/ | grep --extended-regexp 'hbase-[0-9.]+-bin.tar.gz' | grep --extended-regexp --only-matching '[0-9]+\.[0-9]+\.[0-9]+' | head -1)
```

Comment:
Since we have pipes in this script, the script should probably have `set -o pipefail` at the top. Details: https://stackoverflow.com/questions/68465355/what-is-the-meaning-of-set-o-pipefail-in-bash-script

otherwise if there's an error along the pipeline the script will keep going and HBASE_VERSION will just be empty. :/ tested with:

```
set -e

HBASE_VERSION=$(curl https://wwasldfkj.net/hbase/stable/ \
    | grep --extended-regexp 'hbase-[0-9.]+-bin.tar.gz' \
    | grep --extended-regexp --only-matching '[0-9]+\.[0-9]+\.[0-9]+' \
    | head -1)

echo 1111111 $HBASE_VERSION  # prints 1111111 

set -o pipefail

HBASE_VERSION2=$(curl https://wwasldfkj.net/hbase/stable/ \
    | grep --extended-regexp 'hbase-[0-9.]+-bin.tar.gz' \
    | grep --extended-regexp --only-matching '[0-9]+\.[0-9]+\.[0-9]+' \
    | head -1)

echo 2222222 $HBASE_VERSION2  # never runs due to set -e combined with pipefail
```